src/scripts/components/menu.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+export default function Menu() {
+  const body = document.querySelector('.page');
+  const menu = document.querySelector('.navigation');
+  const navList = document.querySelector('.navigation__wrapper');
+  const links = document.querySelectorAll('.navigation__nav-link');
+  const closeBtn = document.querySelector('.navigation__close');
```

Comment:
Better variant use `js-` selectors for using in JS logic. 
For example `.js-page .js-navigation`

Reply:
Oleksandr Ihnatov helped me to understand the issue. 
At first, I was confused, but now understand that all your replies are awesome!)